pkg/ingester/ingesterlimiter/limiter.go

```diff
@@ -3,7 +3,7 @@
 // Provenance-includes-license: Apache-2.0
 // Provenance-includes-copyright: The Cortex Authors.
 
-package ingester
+package ingesterlimiter
```

Comment:
what do we think about internal packages? it will make it easier so we don't end up in super weird dependencies where there shouldn't be any (compactor importing ingester package for a utility)

Reply:
I think that they could be a shoot in our leg when we vendor Mimir in GEM.

Let's just be reasonable and raise our hand if we see someone importing `ingestersomething` from `compactorsomething`